config/initializers/new_framework_defaults.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.0 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
+#
+# Read the Guide for Upgrading Ruby on Rails for more info on each option.
+
+# Enable per-form CSRF tokens. Previous versions had false.
+Rails.application.config.action_controller.per_form_csrf_tokens = false
+
+# Enable origin-checking CSRF mitigation. Previous versions had false.
+Rails.application.config.action_controller.forgery_protection_origin_check =
```

Comment:
Um - "we're not on Ruby 2.4 *yet*?!?!?!"  Ruby 2.4 only came out on December 25, 5 days ago!!!

I think we should move to Rails 5 *first*, as that's better-tested.  Rails 5 obviously works on Ruby 2.3.*.  Once that's settled we can move Ruby.  It appears to me that Ruby 2.4.0 is a relatively quiet change, but I'd like to make 1 change at a time...! :-).